### DISCUSSION TITLE
best practices for setting the created_by id in tables?

### DISCUSSION THREAD

**Discussion by 0radek** [supabase/supabase#4304] [Questions]
When I build my own APIs, I can get user info from a auth token or something and I'd use that id to associate a new record's `created_by` id to the user.

With something like supabase where you do this stuff client-side, do I just store the current user info and pass in a explicit `created_by` id to whatever record I'm trying to make? For example to make a Post with a title, description, and created_by, send a record `{ title, description, created_by }` where the created_by Id is explicit?

Thanks, sry for basic question..

**Comment by VictorPeralta** [comment]
Yes, you can do that. Also enable RLS for instert and check that the logged in user matches the created_at field for the new role, otherwise anyone could pass any user's id. 

Another option would be to get this id from the logged in user in the database, in a function which you can call from the client.
